app/config/import.js

```diff
@@ -60,3 +60,15 @@ exports.getDefaultConfig = () => {
   }
   return defaultConfig;
 };
+
+exports.updateState = function(path, data) {
+  _write(path, data);
+};
+
+exports.importState = function(path) {
+  try {
+    return readFileSync(path, 'utf8');
+  } catch (err) {
+    return 0;
+  }  
+};
```

Comment:
Add an empty line after this one, all files should end with a blank line to make git diffs better 🙂